### ISSUE TITLE
Registration page. Appears without style.

### ISSUE THREAD

**Issue by olairmarcos** [intelliants/subrion#316]
![2017-01-23_193235](https://cloud.githubusercontent.com/assets/8938718/22223789/e3c4a2a0-e19a-11e6-8b83-97a9d8ea8d68.jpg)

Thanks for listening.

**Comment by olairmarcos** [comment]
See also: /panel/configuration/miscellaneous/
![2017-01-23_232809](https://cloud.githubusercontent.com/assets/8938718/22230306/89f8a52a-e1bb-11e6-8a13-8f8f6133fb4c.jpg)

Thanks for listening.

**Comment by vbezruchkin** [comment]
Hi,

it seems you have issues with captcha. It's not displayed on both pages. Could you please confirm you use this installation on the same server you used before and you didn't have the same issue. 

Working fine 
![image](https://cloud.githubusercontent.com/assets/2760455/22233287/8c60e6d4-e219-11e6-8dd4-c12b12c47af8.png)

**Comment by olairmarcos** [comment]
Today (24/01) I downloaded and a new installation Subrion4.1, it is working perfect. I think the problem was related to captcha. Good job.
Thanks for listening.

**Comment by vbezruchkin** [comment]
Yep, looks like a captcha issue. Thanks for reporting.

BTW, check gallery field in admin panel ;) You will be surprised.

**Comment by olairmarcos** [comment]
Hello,
Beautiful work!!
We tested with a new subrion installation, dropzone works fine in the admin panel.
We created a new gallery field and added it to the profile.
A problem appears when we edit the profile on the frontpage, we delete some pictures, the dropzone box does not appear again. When saving the profile again in frontpage, the gallery of images does not appear.
If you load more images through the frontpage, all images saved in the gallery disappear.
The problem is in adding and deleting images by the frontpage on the website.
Thanks for listening.
